Version.tsx

```diff
@@ -0,0 +1,6 @@
+// version.tsx
+export const VERSION = "2.0.0a";
+
+export function getVersion() {
+  return VERSION;
+}

```

Comment:
If we can export const simply then we don't need this function. Just import VERSION from ../path_to_version.tsx and you can use VERSION as variable